### DISCUSSION TITLE
Trying to display png on 320x240 display.

### DISCUSSION THREAD

**Discussion by DrTautology** [russhughes/st7789_mpy#132] [Q&A] [answered]
No issues running the "alien" png program. However if I attempt to just display my png, it looks like it is only represented by a single pixel on the display. I've tried a few different sizes for the png but nothing has worked. Ideally I need the png to fill the screen so I've tried sizing it to match the screen resolution, but no luck.

**Comment by russhughes** [comment]
I will look into this.

**Reply by DrTautology** [reply]
I appreciate it. This is basically the last piece I need for my project.

**Reply by russhughes** [reply]
There is definitely an issue with the .png method.  I'm working on it.

**Reply by russhughes** [reply, marked as answer]
I reworked the PNG code so it should behave much better.

**Reply by DrTautology** [reply]
Thank you so much! I will test it tonight!

**Reply by DrTautology** [reply]
It works beautifully! Thank you again friend. Please let me know if I can provide you with a donation as a show of gratitude.
